### DISCUSSION TITLE
Local development auth.user table doesn't exist

### DISCUSSION THREAD

**Discussion by KrisCoulson** [supabase/supabase#1023] [Questions] [answered]
Am I missing something? I know this feature just came out but I followed the instructions and looking in the .supabase files it seems that the tables should be created. When connected to the database I don't see the Auth Users table 

When trying to use auth service locally to sign up a user receiving a 500 error msg: "Database error finding user" 

One suggestion I found in the supabase issues the auth.users might have been dropped so I thought maybe it was never created. Did I miss a step after I created `supabase init` and `supabase start`
https://github.com/supabase/supabase/issues/663#issuecomment-773967583

**Comment by kiwicopple** [comment, marked as answer]
Sounds like a bug - very sorry @KrisCoulson - there were a lot of last-minute changes before the launch so perhaps we broke something. Let's discuss in this issue: https://github.com/supabase/cli/issues/9

**Reply by KrisCoulson** [reply]
@kiwicopple no problem! I followed up in that issue after I found it through a little searching.
